Publisher/nl/email-editor-drag-and-drop-templates.md

```diff
@@ -25,16 +42,12 @@ Er zijn verschillende blok-opties voor je nieuwsbrief:
 | Video                 | Optie om een video toe te voegen                                                                        |
 | Sociaal               | Optie om social media toe te voegen                                                                     |
 | Banner                | Optie om een banner toe te voegen                                                                       |
-| Timer                 | Optie om een aftel-timer toe te voegen                                                                  |
```

Comment:
De timer is GEEN amp-blok volgens mij

Reply:
Timer blok wordt in eerste instantie niet getoond. Dus weggehaald.